app/scripts/scripts.js

```diff
@@ -1,17 +1,28 @@
-/* For picking letters from alphabet keypad. Returns the
-   letter chosen or "already picked"
-*/
-function pickLetter(letter) {
+(function () {
```

Comment:
I moved verify() out of the closure and called it from inside the closure. Does that work?